juniper_official/oam/ethernet-oam-lfm.rule

```diff
@@ -67,15 +85,15 @@ healthbot {
                     then {
                         status {
                             color green;
-                            message "$lfm-interface-name LFM discovery-state: $lfm-discovery-state; lfm-status: $lfm-status; lfm-peer-address: $lfm-peer-address";
+                            message "$lfm-interface-name LFM discovery-state: $lfm-discovery-state; lfm-status: $lfm-status";
```

Comment:
If peer address is not used anywhere can we remove the field? Please confirm with Venakt.